docs/Configuration/Jira.md

```diff
@@ -7,25 +7,44 @@ description: Config UI instruction for Jira
 Visit config-ui: `http://localhost:4000`.
 
 ### Step 1 - Add Data Connections
+
 ![jira-add-data-connections](/img/ConfigUI/jira-add-data-connections.png)
 
 #### Connection Name
+
 Name your connection.
 
 #### Endpoint URL
+
 This should be a valid REST API endpoint
-   - If you are using Jira Cloud, the endpoint will be `https://<mydomain>.atlassian.net/rest/`
-   - If you are self-hosting Jira v7+ or v8+, the endpoint may look like `https://jira.<mydomain>.com/rest/`
-The endpoint url should end with `/`.
+
+- If you are using Jira Cloud, the endpoint will be `https://<mydomain>.atlassian.net/rest/`
+- If you are self-hosting Jira v7+ or v8+, the endpoint may look like `https://jira.<mydomain>.com/rest/`
+  The endpoint url should end with `/`.
 
 #### Username / Email
+
 Input the username or email of your Jira account.
 
 #### Password
+
 - If you are using Jira Cloud, please input the [Jira personal access token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html).
 - If you are using Jira Server v7+ or v8+, please input the password of your Jira account.
 
+#### Auth Token
+
+When users access Jira API using Jira Auth Token, they may encounter access restrictions if their token does not have sufficient permissions. This is typically caused by insufficient scope or role settings for the Jira Auth Token.
+
+To solve this issue, you can try the following:
+
+- First, check if your Jira Auth Token has sufficient permissions, including the required scope and role. You can set these when creating the token, and incorrect settings will result in restricted access.
```

Comment:
The problem here is that users don't know how to tell if a Jira Auth Token is insufficient for DevLake until they start running pipelines.